lms/views/content_item_selection.py

```diff
@@ -26,6 +27,7 @@ def should_canvas_oauth(request):
 
 @view_config(route_name='content_item_selection', request_method='POST')
 @lti_launch()
+@create_h_user
```

Comment:
I'm creating users in h both when an assignment is launched (the `@create_h_user` in `lti_launches.py` below) and when our app gets launched for selecting a document to annotate during assignment creation (that's this `@create_h_user` here).

There's also this ["module item configuration" view](https://github.com/hypothesis/lms/blob/master/lms/views/module_item_configurations.py) but I don't know what that is and how you actually use it in Canvas so not sure if we want to or can create users then too.